src/Controls/src/Core/Window/Window.cs

```diff
@@ -64,6 +64,14 @@ public partial class Window : NavigableElement, IWindow, IToolbarElement, IMenuB
 		public static readonly BindableProperty TitleBarProperty = BindableProperty.Create(
 			nameof(TitleBar), typeof(TitleBar), typeof(Window), null, propertyChanged: TitleBarChanged);
 
+		/// <summary>Bindable property for <see cref="IsMinimizable"/>.</summary>
+		public static readonly BindableProperty IsMinimizableProperty = BindableProperty.Create(nameof(IsMinimizable),
+			typeof(bool), typeof(Window), defaultValue: true);
+
+		/// <summary>Bindable property for <see cref="IsMaximizable"/>.</summary>
+		public static readonly BindableProperty IsMaximizableProperty = BindableProperty.Create(nameof(IsMaximizable),
+			typeof(bool), typeof(Window), defaultValue: true);
```

Comment:
I just want to check if this works if the app is a maximized app, and then they want to prevent the restore. aka, some maxamized-only app.

Is there a different name that represents the middle button?

Reply:
Seems everyone just uses the maximize words.